Yona/Yona/AppDelegate.swift

```diff
@@ -41,6 +50,27 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
     }
 
+    //MARK: User Methods
+    func getScreenNameToDisplay() -> UINavigationController{
+        
+        
+        let viewName = getViewControllerToDisplay("ScreenToDisplay") as! String
```

Comment:
Should use a closure, not force unwrap if possible:
if let viewName = getViewControllerToDisplay("ScreenToDisplay") as? String {
 ///then put switch statement in here
}